### PR TITLE
myimprovements api integration and added hindi translation 

### PR DESCRIPTION
**Pull request by farhanp1502**

### CODE REVIEW

File: src/app/home/home.page.ts [hunk 1]
```diff
@@ -106,20 +111,17 @@ export class HomePage {
...
+      fetchDataImprovements(improvements),
       fetchData(urls.spotlight),
-      fetchData(urls.recommendation),
     ])
       .pipe(
-        map(([improvementRes, spotlightRes, recommendationRes]: any) => {
-          this.myImprovements = improvementRes.result.data;
-          this.improvementsCount = improvementRes.result.count;
+        map(([improvementRes, spotlightRes]: any) => {
+          this.myImprovements = improvementRes.result[0].data;
```
Comment (5Amogh) [inline src/app/home/home.page.ts:119]: why are we hardcoding index value here?

File: src/app/home/home.page.ts [hunk 2]
```diff
@@ -85,17 +85,22 @@ export class HomePage {
...
+    let  improvements =  `${urlConfig.project.myImprovementsUrl}?&page=${this.page}&limit=${this.limit}&language=en`;
     const urls = {
-      improvements: `${urlConfig.project.myImprovementsUrl}&page=${this.page}&limit=${this.limit}`,
       spotlight: `${urlConfig.project.spotlightUrl}&page=${this.page}&limit=${this.limit}`,
-      recommendation: `${urlConfig.project.recommendationUrl}?page=${this.page}&limit=${this.limit}`,
     };
+    const fetchDataImprovements = (url:any) =>
+      this.baseApiService.post(url,{}).pipe(
+        catchError((err) => {
+          this.toastService.presentToast(err.error.message, 'danger');
```
Comment (5Amogh) [inline src/app/home/home.page.ts:101]: When we show this error msg via toast, will the user be able to understand it? are we sure of it? or should we log this error and show some use friendly msg as toast msg

File: src/app/generic-listing-page/my-improvements-listing/my-improvements-listing.page.ts [hunk 3]
```diff
@@ -47,15 +47,15 @@ export class MyImprovementsListingPage implements OnInit {
...
-    let url = `${urlConfig.project.myImprovementsUrl}&page=${this.page}&limit=${this.limit}&search=&filter=`;
-    this.projectsApiService.get(url).subscribe({
+    let url = `${urlConfig.project.myImprovementsUrl}?&page=${this.page}&limit=${this.limit}&search=&language=en`;
+    this.projectsApiService.post(url,{}).subscribe({
       next: async (response: any) => {
         await this.loaderService.dismissLoading();
         if (response.status == 200) {
           this.myImprovements = this.myImprovements.concat(
-            response.result.data
+            response.result[0].data
```
Comment (5Amogh) [inline src/app/generic-listing-page/my-improvements-listing/my-improvements-listing.page.ts:56]: reason behind index hardcoding?

**Review by 5Amogh — CHANGES_REQUESTED** [review]

File: src/app/home/home.page.ts (hunk 1 above)
Comment (farhanp1502) [reply src/app/home/home.page.ts:119]: in the api response we are getting the response structure like this

File: src/app/home/home.page.ts (hunk 2 above)
Comment (farhanp1502) [reply src/app/home/home.page.ts:101]: we need to request the msg from the products team

File: src/app/generic-listing-page/my-improvements-listing/my-improvements-listing.page.ts (hunk 3 above)
Comment (farhanp1502) [reply src/app/generic-listing-page/my-improvements-listing/my-improvements-listing.page.ts:56]: same response structure

**Pull request merged**